module/VuFind/src/VuFind/Log/Handler/Office365Handler.php

```diff
@@ -22,47 +22,48 @@
  *
  * @category VuFind
  * @package  Error_Logging
- * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Sambhav Pokharel <sambhav.pokharel@gmail.com>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Site
  */
 
-namespace VuFind\Log\Writer;
+namespace VuFind\Log\Handler;
 
 use Laminas\Http\Client;
 
 /**
- * This class extends the Laminas Logging to send errors to Office365 webhooks.
+ * This class extends the Monolog Logging to send errors to Office365 webhooks.
  *
  * @category VuFind
  * @package  Error_Logging
- * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Sambhav Pokharel <sambhav.pokharel@gmail.com>
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Site
  */
-class Office365 extends Post
+
+class Office365Handler extends PostHandler
 {
     /**
      * The title for generated cards.
      *
      * @var string
      */
-    protected $title;
+    protected string $title;
 
     /**
      * Constructor
      *
      * @param string $url     URL to open as a stream
      * @param Client $client  Pre-configured http client
-     * @param array  $options Optional settings (may contain 'channel' for the
-     * Slack channel to use and/or 'name' for the username messages are posted under)
+     * @param array  $options Optional settings
      *
      * @throws \Exception
      */
-    public function __construct($url, Client $client, array $options = [])
+    public function __construct(protected string $url, protected Client $client, array $options = [])
```

Comment:
Since you are passing `$url` and `$client` to the parent constructor below, you don't need to use property promotion here; that will create a duplicate local definition of the properties. Functionally, I don't think it makes a significant difference, but it's simpler and more efficient to do it the traditional way in this case.

```suggestion
    public function __construct(string $url, Client $client, array $options = [])
```